### PR TITLE
Display error tooltip in ActivityIndicator

### PR DESCRIPTION
**Pull request by meriadec**
error

### CODE REVIEW

File: src/components/TopBar/ActivityIndicator.js [hunk 1]
```diff
@@ -123,6 +126,23 @@ class ActivityIndicatorInner extends PureComponent<Props, State> {
...
     )
+
+    if (error) {
+      return (
+        <Tooltip
+          tooltipBg="alertRed"
+          render={() => (
+            <Box fontSize={4} p={2} style={{ maxWidth: 250 }}>
+              <TranslatedError error={error} />
+            </Box>
```
Comment (gre) [inline src/components/TopBar/ActivityIndicator.js]: could use a variable as well because the inner don't change

Comment (gre) [reply src/components/TopBar/ActivityIndicator.js]: actually maybe you don't need a render function , just a dom , like children ?

Comment (gre) [reply src/components/TopBar/ActivityIndicator.js]: it's ok for now, i'll merge

Comment (meriadec) [reply src/components/TopBar/ActivityIndicator.js]: +1 we could refactor all tooltips passing nodes directly

**Pull request merged**